Pipfile

```diff
@@ -4,10 +4,6 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pylint = "*"
```

Comment:
The [dev-packages] were removed --- are they not required or needed on this file anymore?

Reply:
Added back dev packages back in.